itests/fevm_test.go

```diff
@@ -7,14 +7,17 @@ import (
 	"encoding/binary"
 	"encoding/hex"
```

Comment:
Why do we have such a large diff in this file ? Can we get rid of all the whitespace changes and only change the test code we need to ? Reviewing diffs like this is tricky.

Reply:
It's a linter fix, everywhere there is whitespace after `//`

Reply:
if this is from your editor doing this automatically then it's typically good practice to disable that--it's fine to make lint fixes but large diffs like that should be done as a separate PR and not come in with a PR where the changes are unrelated, or at least make them minimal (e.g. it's OK to fix up some of these around the site where you're changing things, but usually not the whole file if you can help it.)

Reply:
So generally I try to do `go fmt` in files after changing things that's why sometimes I get large diff, anyway, I will disable it and revert these whitespace changes so we can keep it clean.